dtfs-central-api/src/v1/swagger-definitions/utilisation-reports/utilisation-report.js

```diff
@@ -8,13 +8,31 @@
  *         type: string
  *         example: '9'
  *         description: Bank id. Separate MongoDB _id
- *       month:
- *         type: integer
- *         example: 1
- *         description: Month of utilisation report indexed at 1
- *       year:
- *         type: integer
- *         example: 2021
+ *       reportPeriod:
+ *         type: object
+ *         properties:      
```

Comment:
Some extra spaces at the end of the line here to be removed.

Reply:
I can't see any empty spaces 😕

Reply:
Oh I'm guessing they were removed 😆